apps/internal/base/internal/storage/storage.go

```diff
@@ -40,8 +40,6 @@ type TokenResponse struct {
 	Account      shared.Account
 }
 
-// TODO(someone): This thing does not expire tokens.
```

Comment:
was this resolved?

Reply:
Yes, MSAL does not technically have a mechanism to remove old expired access tokens, we just overwrite it if it is expired, Refer: https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/dev/msal/application.py#L1025
We do have removeAccount which will explicitly signout and forget all the accessTokens, RefreshTokens, IDTokens assosciated with that account. This is what the PR adds.